default/scripting/techs/ships/Weapons.focs.txt

```diff
@@ -9,6 +9,19 @@ Tech
         Item type = ShipPart name = "SR_WEAPON_1_1"
         Item type = ShipPart name = "GT_TROOP_POD"
     ]
+    effectsgroups =
+        EffectsGroup
+            scope = And [
+                Ship
+                OwnedBy empire = Source.Owner
+                DesignHasPart  name = "SR_WEAPON_1_1"
+            ]
+            accountinglabel = "SR_WEAPON_1_1"
+            effects = [
+                SetMaxCapacity partname = "SR_WEAPON_1_1" value = Value + 5
```

Comment:
should use a lookup for the relevant part's stats, rather than re-hard-coding here.